solr/core/src/java/org/apache/solr/search/facet/FacetFieldProcessorByHashDV.java

```diff
@@ -187,6 +187,11 @@ protected Comparable parseAndAddGap(Comparable value, String gap) throws ParseEx
       throw new UnsupportedOperationException();
     }
 
+    @Override
+    protected Comparable toSortableDocValue(Comparable calcValue) {
+      return calcValue; // TODO
+    }
+
```

Comment:
How critical is implementing this? What breaks if we leave this as a TODO?

Reply:
This specifically is for `TermOrdCalc` which applies to string values. It needs more though. Right now, I have precondition checks that fail with bad request if you try to doc value range facet on anything other than single valued numerics.